### DISCUSSION TITLE
Please Add AssetManager Config for HTTP Method

### DISCUSSION THREAD

**Discussion by at-codes** [GrapesJS/grapesjs#4984] [Ideas]
Can [this](https://github.com/GrapesJS/grapesjs/blob/dev/src/asset_manager/view/FileUploader.ts#L176) get updated to be a config? I'd like to set it to a put instead a post.

**Comment by artf** [comment]
Added [here](https://github.com/GrapesJS/grapesjs/commit/6184b6c86c92af2cc30fe79665351747b346bfa2)
